packages/ibm-products/src/components/UserAvatar/UserAvatar.js

```diff
@@ -58,8 +58,10 @@ export let UserAvatar = React.forwardRef(
       // The component props, in alphabetical order (for consistency).
       backgroundColor,
       className,
+      name,
       /* TODO: add other props for UserAvatar, with default values if needed */
       renderIcon = defaults.renderIcon,
+      size = 'md',
       tooltipText = defaults.tooltipText,
```

Comment:
I think we can move `size` to defaults.

I missed this in the previous PRs, but we probably should not set a default for `tooltipText` since this is optional. The default here should only be set in the story.

```suggestion
      size = 'md',
      tooltipText = defaults.tooltipText,
```